<ui version="4.0" >
 <class>FontExtractorDialog</class>
 <widget class="QDialog" name="FontExtractorDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Font extractor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Select PDF</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="docPath" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>File name</string>
        </property>
        <property name="buddy" >
         <cstring>browsePDF</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="browsePDF" >
        <property name="text" >
         <string>&amp;Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Fonts</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" colspan="3" >
       <widget class="QListWidget" name="fontList" />
      </item>
      <item row="1" column="0" >
       <widget class="QLineEdit" name="outputDir" />
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="browseDir" >
        <property name="text" >
         <string>&amp;Select Dir</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="extractButton" >
        <property name="text" >
         <string>&amp;Extract</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="pushButton" >
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>FontExtractorDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>452</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel" >
     <x>458</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
